starfish/spots/gaussian.py

```diff
@@ -6,30 +6,32 @@
 from showit import image
 from skimage.feature import blob_log
 
-from starfish.munge import stack_to_list, gather
+from starfish.munge import gather
 
 
 class GaussianSpotDetector:
-    def __init__(self, stack, blobs, measurement_type='max'):
+    def __init__(self, stack):
```

Comment:
ditto here.  the detector should be parameterized in the scope of the constructor.  data is passed in.

imagine that you have a large number of images, and you set up the detector and do any heavy work involved with the setup there.  you can keep feeding in stuff, and extract out what you want.

Reply:
Having said that, I am not opposed to you landing this, and I'll just add this to the list of cleanup tasks I'm going to do as I go through the repo.